src/components/RecordedVideo.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { isSafari } from 'react-device-detect'
+
+export const RecordedVideo = (props: { recordedChunks: BlobPart[] }) => {
+  const [videoUrl, setVideoUrl] = useState<string>('')
+  const videoRef = useRef<HTMLVideoElement>(null)
+  const [isVideoStarted, setIsVideoStarted] = useState<boolean>(false)
+  const blob = new Blob(props.recordedChunks, {
+    type: isSafari ? 'video/mp4' : 'video/webm',
+  })
+  const url = URL.createObjectURL(blob)
+
+  useEffect(() => {
+    setVideoUrl(url)
+  }, [])
+
+  const startVideo = () => {
+    if (videoRef.current) {
+      const video = videoRef.current
```

Comment:
こっちのほうがちょっとだけ短くなる気がします
```suggestion
    const video = videoRef.current
    if (video) {
```